packages/react-components/react-search-preview/src/components/SearchBox/useSearchBoxStyles.styles.ts

```diff
@@ -90,6 +93,7 @@ const useDismissStyles = makeStyles({
  * Apply styling to the SearchBox slots based on the state
  */
 export const useSearchBoxStyles_unstable = (state: SearchBoxState): SearchBoxState => {
+  useInputStyles_unstable(state);
```

Comment:
I think the regressions we're seeing are due to this function being called here.

This happens because we're effectively appending the input classnames after the searchbox classnames in this scenario, as `state.[slot].className` already includes the input classnames after this call.

To eliminate the regressions you should move this call just before you return state at the end of this function. Something similar happens in `ToggleButton` and you can take a look there to see we solved it by doing the same thing.